Package.swift

```diff
@@ -24,14 +24,15 @@ let package = Package(
         .library(name: "SpeziLLMOpenAI", targets: ["SpeziLLMOpenAI"])
     ],
     dependencies: [
-        .package(url: "https://github.com/MacPaw/OpenAI", .upToNextMinor(from: "0.2.5")),
+        .package(url: "https://github.com/MacPaw/OpenAI", revision: "35afc9a6ee127b8f22a85a31aec2036a987478af"),    // No new release from MacPaw, use latest commit to main until release is tagged
```

Comment:
Sadly, MacPaw/OpenAI didn't tag a new release yet. I encouraged it via https://github.com/MacPaw/OpenAI/pull/150#issuecomment-1933437896

Therefore, via https://developer.apple.com/documentation/packagedescription/package/dependency/package(url:revision:), our SpeziLLM package is now depending on the newest commit hash on `main` of MacPaw/OpenAI

Reply:
As far as I'm seeing, this also shouldn't result in any versioning issues, at least the docs don't mention this limitation: https://developer.apple.com/documentation/packagedescription/package/dependency/package(url:revision:)